tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
remove this comment, please
it looks extra

Reply:
fixed